src/main/kotlin/com/learnspigot/bot/entity/Giveaway.kt

```diff
@@ -0,0 +1,27 @@
+package com.learnspigot.bot.entity
+
+import dev.morphia.annotations.Entity
+import dev.morphia.annotations.Id
+import java.time.Instant
+
+@Entity("giveaways")
+data class Giveaway(
+    @Id val id: String,
+    val channelId: String = "",
```

Comment:
Surely this shouldn't need a default value. I think we only need default values for migrated values

Reply:
> Surely this shouldn't need a default value. I think we only need default values for migrated values

If they don't have a default value findGiveaway starts yelling at me :(

Reply:
Ah ok

Reply:
There might and probably is a smarter way, but I couldn’t figure it out. As a precaution any “broken” giveaways are removed on startup tho so they won’t persist